src/ompl/robots.cpp

```diff
@@ -3172,6 +3172,26 @@ void load_motion_primitives_new(const std::string &motionsFile,
     // motions[idx].last_state_translated = motions[idx].traj.states.back();
   }
 }
+// add N motions to motion_out. It starts adding from [current_size,....+N]
+void motion_to_motion(std::vector<Motion> &robot_motions,
```

Comment:
don't really understand what the function is supposed to do from the name and comment